mscore/exportdialog.ui

```diff
@@ -23,108 +23,55 @@
    <string>Export</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1">
-   <item row="0" column="0" colspan="2">
-    <layout class="QHBoxLayout" name="horizontalLayout">
-     <property name="topMargin">
-      <number>8</number>
-     </property>
-     <property name="bottomMargin">
-      <number>8</number>
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
```

Comment:
Why are the margins changed? I don't think this is intended... please undo this (remove these 12 lines).